crates/sui-sdk/src/apis.rs

```diff
@@ -148,13 +434,33 @@ impl ReadApi {
             .await?)
     }
 
+    /// Return the [SuiCommittee] information for the provided `epoch`, or an error upon failure.
+    ///
+    /// The argument `epoch` is either a known epoch id or `None` for the last epoch.
```

Comment:
I would describe it as the "current" epoch, rather than the "last" which could be taken to mean the previous epoch.